src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-form/cephfs-subvolume-form.component.html

```diff
@@ -43,6 +43,26 @@
           </div>
         </div>
 
+          <!--Subvolume Group name -->
+          <div *ngIf="!this.isEdit"
```

Comment:
we shouldn't hide this when we are editing, instead the select should be disabled but shows the subvolume group which the subvolume is in.